default/stringtables/en.txt

```diff
@@ -7164,14 +7177,16 @@ Sly
 
 SP_SLY_GAMEPLAY_DESC
 '''Friendly overlooked gasbags.
-Only live an [[PT_GASGIANT]]s.
-[[encyclopedia SELF_SUSTAINING_SPECIES_TITLE]]
+Greatly prefer [[PT_GASGIANT]]s.
+
+[[encyclopedia GASEOUS_SPECIES_TITLE]]
 '''
 
 SP_SLY_DESC
 '''Description:
 The Sly live in gas giants. The gas giants are so foreign to most species in the universe that, for most of their history, they have been overlooked.
 Not being able to build devices, they couldn't leave their homeworld. Recently, they were able to obtain necessary equipment and technology for spacefaring from traders in exchange for their knowledge of history.
+Keeping a strict diet a Sly crew on gas giants actually can help in refinining fuel for their ship.
```

Comment:
this sentence is borderline incoherent without a comma or rewording / reordering